openid-sharedsignals-framework-1_0.md

```diff
@@ -0,0 +1,2169 @@
+      "subject": {
+        "format": "email",
+        "email": "foo@example2.com"
+      },
+      "event_timestamp": 1600975810,
+      "claims": {
+         "role": "ro-admin"
+      }
+    }
+  }
+}
+~~~
+{: #subject-properties-ex title="Example: SET Containing a SSE Event with a Simple Subject and a Property Member"}
+
+~~~ json
+{
+  "iss": "https://myservice.example3.com/",
+  "jti": "756E69717565206964656E746966696534",
+  "iat": 15203800012,
+  "aud": "636C69656E745F6324",
+  "events": {
+    "https://schemas.openid.net/secevent/caep/event-type/token-claims-change": {
+    "subject": {
+        "format": "catalog_item",
+        "catalog_id": "c0384/winter/2354122"
+      },
+      "event_timestamp": 1600975810,
+      "claims": {
+         "role": "ro-admin"
+      }
+    }
+  }
+}
+~~~
+{: #subject-custom-type-ex title="Example: SET Containing a SSE Event with a Proprietary Subject Identifier Format"}
+
+# Transmitter Configuration Discovery {#discovery}
+
+This section defines a mechanism for Receivers to obtain Transmitter
+configuration information.
+
+## Transmitter Configuration Metadata {#discovery-meta}
+
+Transmitters have metadata describing their configuration:
+
+issuer
+
+> REQUIRED. URL using the https scheme with no query or fragment component
+  that the Transmitter asserts as its Issuer Identifier. This MUST be identical
+  to the iss claim value in Security Event Tokens issued from this Transmitter.
+
+jwks_uri
+
+> REQUIRED. URL of the Transmitter's JSON Web Key Set {{RFC7517}} document.
+  This contains the signing key(s) the Receiver uses to validate signatures from
+  the Transmitter.
+
+delivery_methods_supported
+
+> RECOMMENDED. List of supported delivery method URIs.
+
+configuration_endpoint
+
+> OPTIONAL. The URL of the Configuration Endpoint.
+
+status_endpoint
+
+> OPTIONAL. The URL of the Status Endpoint.
+
+add_subject_endpoint
+
+> OPTIONAL. The URL of the Add Subject Endpoint.
+
+remove_subject_endpoint
+
+> OPTIONAL. The URL of the Remove Subject Endpoint.
+
+verification_endpoint
+
+> OPTIONAL. The URL of the Verification Endpoint.
+
+critical_subject_members
+
+> OPTIONAL. List of member names in a Complex Subject which, if present in
+  a Subject Member in an event, MUST be interpreted by a Receiver.
+
+TODO: consider adding a IANA Registry for metadata, similar to Section 7.1.1 of
+{{OAUTH-DISCOVERY}}. This would allow other specs to add to the metadata.
+
+## Obtaining Transmitter Configuration Information
+
+Using the Issuer as documented by the Transmitter, the Transmitter Configuration
+Information can be retrieved.
+
+Transmitters supporting Discovery MUST make a JSON document available at the
+path formed by inserting the string `/.well-known/sse-configuration` into the
+Issuer between the host component and the path component, if any. The syntax
+and semantics of `.well-known` are defined in {{RFC5785}}.  `sse-configuration`
```

Comment:
I was hoping that these back-ticks would lead to quotes in the txt file to address the comments you made [here](https://github.com/openid/sse/pull/9#discussion_r1024664360), which I addressed in this [pull request](https://github.com/openid/sse/pull/35) that hacks around the issue when converting from XML to txt. Unfortunately, going from markdown to text does not seem to fix the issue. We'll need to update that hacky pull request to fix this now that we're using markdown as the source.

Reply:
changed all back quotes to double quotes in the MD source.